### PR TITLE
Exposed Native Engine capabilities to Babylon.js, fixes morph targets on Babylon Native Android

### PR DESCRIPTION
**Pull request by SergioRZMasson**
## Overview

Morph targets are currently failing on Babylon Native when running on Android since bgfx does not support Texture2D arrays for Android API BGFX_CONFIG_RENDERER_OPENGLES lower than 31 (currently set to 30). The appropriate way of handling this type of problem would be to check bgfx capabilities and handle it appropriately on the JS side by setting ```canUseTextureForTargets``` property at ```MorphTargetManager```. This PR exposes all capabilities info provided by the Native Engine so they can be accessed by the js NativeEngine and appropriately handled by Babylon.js 

## Aditions

- Exposed Native engine caps to javascript. 
- After this PR gets merged the following [Babylon Native PR](https://github.com/BabylonJS/BabylonNative/pull/1075) can be merged.
- Fixes [Android Morph Targets](https://github.com/BabylonJS/BabylonNative/issues/1049) issue on Babylon Native.

### CODE REVIEW

**Review by RaananW — APPROVED** [review]
Waiting for @sebavan review as well before merging

File: packages/dev/core/src/Engines/webgpuEngine.ts
Comment (deltakosh) [inline packages/dev/core/src/Engines/webgpuEngine.ts:813]: not sure what it means here to have a webglversion (for webgpu)

File: packages/dev/core/src/Engines/nullEngine.ts
Comment (deltakosh) [inline packages/dev/core/src/Engines/nullEngine.ts:155]: not sure what it means here to have a webglversion (for nullengine we do't care)

File: packages/dev/core/src/Engines/webgpuEngine.ts
Comment (SergioRZMasson) [reply packages/dev/core/src/Engines/webgpuEngine.ts]: You are right @deltakosh, this value should be set to 2048 for webgpu. Updated in the latest commit.

File: packages/dev/core/src/Engines/Native/nativeInterfaces.ts
Comment (bghgary) [inline packages/dev/core/src/Engines/Native/nativeInterfaces.ts:113]: Are these all being used?

File: packages/dev/core/src/Engines/nativeEngine.ts
Comment (bghgary) [inline packages/dev/core/src/Engines/nativeEngine.ts:891]: We need to increment the PROTOCOL_VERSION for this change.

File: packages/dev/core/src/Engines/Native/nativeInterfaces.ts
Comment (SergioRZMasson) [reply packages/dev/core/src/Engines/Native/nativeInterfaces.ts:113]: No, we currently only use CAPS_LIMITS_MAX_TEXTURE_LAYERS, but I'm exposing all limits provided by bgfx in case the native engine needs any of them in the future. This way we don't have to change the PROTOCOL_VERSION again.

Comment (bghgary) [reply packages/dev/core/src/Engines/Native/nativeInterfaces.ts:113]: I'm not sure about this. My guess is that you will have to update the protocol version anyways as this list may not translate correctly.

Comment (SergioRZMasson) [reply packages/dev/core/src/Engines/Native/nativeInterfaces.ts]: I've updated the exposed caps to only have the ones we currently require or have a direct equivalent on the js caps.

**Review by bghgary — APPROVED** [review]

**Pull request merged**
